Not sure why the tests are failing here, the test in question is not related to my changes. I am not able to build the aspnet core project, it complains about needing a new version of MSBuild but I have the latest version of VS2019 installed.

```
C:\Code\aspnetcore\eng\tools\RepoTasks\RepoTasks.csproj : error : Version 5.0.100-preview.5.20251.2 of the .NET Core SDK requires at least version 16.5.0 of MSBuild. The current available version of MSBuild is 16.3.0.46305. Change the .NET Core SDK specified in global.json to an older version that requires the MSBuild version currently available.
C:\Code\aspnetcore\eng\tools\RepoTasks\RepoTasks.csproj : error MSB4236: The SDK 'Microsoft.NET.Sdk' specified could not be found.
```

--- a/src/Middleware/Rewrite/test/IISUrlRewrite/MiddleWareTests.cs
+++ b/src/Middleware/Rewrite/test/IISUrlRewrite/MiddleWareTests.cs
@@ -1,4 +1,4 @@
-﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Copyright (c) .NET Foundation. All rights reserved.
 // Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
 
 using System;
@@ -399,7 +399,7 @@ namespace Microsoft.AspNetCore.Rewrite.Tests.UrlRewrite
 
             var response = await server.CreateClient().GetAsync("article/23?p1=123&p2=abc");
 
-            Assert.Equal("/blogposts/article/abc", response.Headers.Location.OriginalString);
+            Assert.Equal("/blogposts/article/abc?p1=123&p2=abc", response.Headers.Location.OriginalString);
         }
 
         [Fact]
@@ -427,7 +427,7 @@ namespace Microsoft.AspNetCore.Rewrite.Tests.UrlRewrite
 
             var response = await server.CreateClient().GetAsync("article/23?p1=123&p2=abc");
 
-            Assert.Equal("/blog/article/23/abc", response.Headers.Location.OriginalString);
+            Assert.Equal("/blog/article/23/abc?p1=123&p2=abc", response.Headers.Location.OriginalString);
         }
 
         [Fact]

